specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -391,6 +391,37 @@
           }
         }
       }
+    },
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/namespaces/{namespaceName}/messagingplan": {
+      "get": {
```

Comment:
I do not see a network payload example - please add "x-ms-examples" for this operation